_schemaV2.graphql

```diff
@@ -4922,6 +4922,14 @@ type Feature {
   subheadline(format: Format): String
 }
 
+# An illustrated link chosen to highlight a Gene from a given GeneFamily
```

Comment:
I asked about this diff [here](https://artsy.slack.com/archives/CP9P4KR35/p1594070048419600). These are valid changes leftover from a previous PR so safe to be merge. (if not already merge by the time this gets reviewed)